src/frontend/tsx/Chessboard.tsx

```diff
@@ -196,6 +196,12 @@ export function Chessboard () {
       fen: (displayIndex == null || displayGame.info.moves == null)
         ? chess.fen()
         : displayGame.info.moves[displayIndex].fen,
+      lastMove: (displayGame == null || displayGame.info.moves == null)
+        ? null
+        : [
+          displayGame.info.moves[displayIndex].from,
+          displayGame.info.moves[displayIndex].to
+        ],
```

Comment:
There's a slight bug here. What we really want is:
- If the display game is `null` (meaning we're looking at the practice board)
    - stored prev move for practice board
- If the display game is not `null`, but there's no moves yet
    - `null`
- If the display game is not `null`, and there are moves, but `displayIndex` is `null`
    - final element in list of moves
- If the display game is not `null`, and there are moves, and `displayIndex` is not `null`
    - `displayIndex`

However, since we don't have a way to track the previous move on the practice board (and it would be surprisingly annoying to introduce it), that case should also be `null`.